joomla/joomla-cms
# [5.1] Implement Schema.org Product type plugin

### Summary of Changes
Implements Schema.org plugin for Product type.


### Testing Instructions
1. Ensure the Product schema plugin is installed (if switching to this branch be sure to use install discovery) and published
2. Edit any Article of your choosing in Content > Articles
3. Under the Schema tab select "Product" Schema type
4. Add whatever product information you would like and save the article
5. View the article on frontend to verify product schema has been output to the head

Your schema output on frontend should be for example similar to the below, which has already been ran through schema test tools and verified (non-critical issues will be reported for missing ratings and can add support for supplying ratings if desired, but seamed weird to hardcode that in).

![image](https://github.com/joomla/joomla-cms/assets/1325443/f5cc4c3e-e265-4b8b-bd3f-d41c441cfe1c)



### Actual result BEFORE applying this Pull Request
No Product Schema.org type available


### Expected result AFTER applying this Pull Request
Product Schema.org type available


### Link to documentations
Please select:
